Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -128,6 +129,33 @@
     },
 ]
 
+
+class RepeatedTimer:
```

Comment:
Not sure why we need this class. The `Timer` class supports args and kwargs. I think you can just use the `Timer` class directly.